File: imagepicker/src/main/java/com/esafirm/imagepicker/features/common/ImageLoaderListener.java
package com.esafirm.imagepicker.features.common;

import com.esafirm.imagepicker.model.Folder;
import com.esafirm.imagepicker.model.Image;

import java.util.List;

public interface ImageLoaderListener {
    void onImageLoaded(List<Image> images, List<Folder> folders);
    void onFailed(Throwable throwable);
}

File: imagepicker/src/main/java/com/esafirm/imagepicker/features/common/ImageLoaderListener.kt
package com.esafirm.imagepicker.features.common

import com.esafirm.imagepicker.model.Folder
import com.esafirm.imagepicker.model.Image

interface ImageLoaderListener {
    fun onImageLoaded(images: List<Image>?, folders: List<Folder>?)
    fun onFailed(throwable: Throwable)
}